.eslintrc.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
         'header/header': ['off']
       }
     }
-  ]
+  ],
+  plugins: ['jsdoc']
```

Comment:
this def introduces a ton of warnings but personally I think it's ok to address these as files are touched, thoughts @michaelgoin ?